modules/apps/frontend-js/frontend-js-components-web/src/main/resources/META-INF/resources/index.d.ts

```diff
@@ -0,0 +1,110 @@
+/**
```

Comment:
So the only items exported in the index.js are...
```js
export {default as Treeview} from './treeview/Treeview';

export {activeLanguageIdsAtom} from './translation_manager/state';
export {default as TranslationAdminModal} from './translation_manager/TranslationAdminModal';
export {default as TranslationAdminSelector} from './translation_manager/TranslationAdminSelector';
```

We should only need declared types for `Treeview`, `activeLanguageIdsAtom`, `TranslationAdminModal`, and `TranslationAdminSelector`.

So I don't think we need to add types for the other items such as `NodeList`

Reply:
Ah, I see, so those components like `NodeList` are exported but they're all used internally, and we only care about components/utilities exported to outside of a specific module, so just the stuff that gets exported in the `index.js` file. Makes sense 👍